libvast/src/system/disk_monitor.cpp

```diff
@@ -85,8 +92,10 @@ disk_monitor(disk_monitor_actor::stateful_pointer<disk_monitor_state> self,
       const auto index_dir
         = std::filesystem::directory_iterator(self->state.dbdir / "index", ec);
       if (ec)
-        return caf::make_error(ec::filesystem_error, "db-directory does not "
-                                                     "exist");
+        return caf::make_error(ec::filesystem_error, //
```

Comment:
Did you mean to have `//` here at the end?

Reply:
This is just forcing a line break because clang-format is stupid.